tests/Unit/QuerySortMergedTest.php

```
@@ -0,0 +1,253 @@
+        parent::setUp();
+
+        $this->wpdb         = Mockery::mock('stdClass');
+        $this->wpdb->prefix = 'wp_';
+
+        $GLOBALS['wpdb'] = $this->wpdb;
+
+        Functions\stubs([
+            'get_transient'    => false,
+            'set_transient'    => true,
+            'delete_transient' => true,
+            'wp_json_encode'   => static fn($data, ...$opts) => json_encode($data),
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$opts'. (undefined)

(UnusedFormalParameter)